src/apply.ts

```diff
@@ -227,23 +115,45 @@ export async function apply(options: ReadOptions): Promise<number> {
 		);
 	}
 
+	type RemoteImage = {
+		registry: string;
+		repository: string;
+		image_name: string;
+		content_hash: string;
+		service: string;
+		status: string;
+	};
+
+	const remoteImages: RemoteImage[] = bundle.manifest.release_image!.map(
```

Comment:
If “release_image” is null, force unwrapping the optional (with “!”) will crash the process. Only use “!” when you have proved that the value is not null (and recorded that proof in a comment). Can you prove this won’t ever be null?

Reply:
I added a check after the reading the bundle to make sure that this property exists.  We will add more checks in a separate PR.